### ISSUE TITLE
Ephem.from_horizons can't handle multiple Time values

### ISSUE THREAD

**Issue by jianyangli** [NASA-Planetary-Science/sbpy#100]
Below code generates an error message:
```
from sbpy.data import Ephem
from astropy.time import Time
epochs = Time(['2015-01-01', '2015-01-02'])
eph = Ephem.from_horizons('Ceres', epochs=epochs)
```

The problem is that `.from_horizons` doesn't handle `epochs` value correctly when it contains multiple values.  There are also other bugs for the case `isinstance(epochs, (list, tuple, ndarray))`.

**Comment by jianyangli** [comment]
I think we should either describe what data type it accepts clearly in the docs, or make it truly flexible. 
But I just noticed that Michael did note it in the code that this input needs to be modified.  So the potential problems here are in his radar already.

**Comment by jianyangli** [comment]
I'm closing this issue now as the problem has been resolved with PR #106
